source/content/drupal-integrated-composer.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Integrated Composer
+description: Learn how to deploy a Drupal site with integrated Composer
+tags: [composer, workflow]
+categories: [drupal]
+contributors: [edwardangert]
+searchboost: 150
+reviewed: "2020-07-21"
+---
+
+Pantheon is testing a release of Drupal with integrated Composer and is opening the feature to users for Limited Availability testing. Integrated Composer lets you deploy your site on Pantheon with one-click updates for both upstream commits and Composer dependencies, while still receiving upstream updates.
+
+## Get access to Integrated Composer
+
+Contact [Support](/support). If you are not part of an organization already, your CSE will add you to a test organization from which you can deploy the upstream. If you are already part of an organization, the feature can be enabled organization-wide.
```

Comment:
Integrated Composer is currently limited to customers with a Pantheon organization. Contact [Support](/support) to request access. Once enabled for your organization, you can enable it for a custom upstream via pantheon.upstream.yml or for individual sites via pantheon.yml.

Reply:
updated to match the first part
I'd like to include the second part in the next section, would it be a better step 3 there?
> enable it for a custom upstream via pantheon.upstream.yml or for individual sites via pantheon.yml